modules/webclient/templates/i2b2_config_data.js.erb

```diff
@@ -0,0 +1,19 @@
+{
+	urlProxy: "index.php",
+	urlFramework: "js-i2b2/",
+	//-------------------------------------------------------------------------------------------
+	// THESE ARE ALL THE DOMAINS A USER CAN LOGIN TO
+	lstDomains: 
+	[
+		<% @domains.each do |domain| -%>
+		{
+			domain:        "<%= domain['domain'] %>",
+			name:          "<%= domain['name'] %>",
+			urlCellPM:     "<%= domain['urlCellPM'] %>",
+			allowAnalysis: "<%= domain['allowAnalysis'] %>",
+			debug:         "<%= domain['debug'] %>"
+		},
+		<% end -%>
```

Comment:
It suppose it's possible you would want to deploy a webclient connected to several domains, some of which you may not even manage. In any case, the class accepting extra configuration for the domains makes sense (but it would be a parameter of `i2b2::webclient_root` then). In any case, it should at least default to the settings on `i2b2::params`.